src/psyclone/psyir/symbols/datatypes.py

```diff
@@ -294,17 +300,18 @@ def intrinsic(self):
     @property
     def is_allocatable(self) -> bool:
         '''If we have enough information in the partial_datatype,
-        determines if this data type is allocatable, and False otherwise.
+        determines whether this data type is allocatable or not.
+        If it is unknown, it will return None. Note that atm PSyclone
+        only supports allocatable arrays.
```

Comment:
Took me a while to get to grips with this statement (because I thought "but PSyclone supports static arrays too"). Perhaps add emphasis to "arrays"?

Reply:
I reworded it a bit, and added emphasis.

Reply:
Oh yes, that helps. Thanks.